.rubocop.yml

```diff
@@ -73,6 +73,7 @@ Metrics/CyclomaticComplexity:
     - 'lib/orangelight/browse_lists/call_number_csv.rb'
 
 Metrics/LineLength:
+  Max: 250
```

Comment:
Could we use an inline exception around the one test instead of changing this globally?

Reply:
Sure, I can adjust this.

Reply:
I've reverted the commits, to be addressed as part of #2285.